modules/api/src/main/java/net/countercraft/movecraft/events/ItemHarvestEvent.java

```diff
@@ -0,0 +1,39 @@
+package net.countercraft.movecraft.events;
+
+
+import net.countercraft.movecraft.craft.Craft;
+import org.bukkit.event.HandlerList;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+/**
+ * Called when one or more blocks are harvested by a moving craft
+ */
+public class ItemHarvestEvent extends CraftEvent {
+    private List<ItemStack> harvested;
+    private static final HandlerList HANDLER_LIST = new HandlerList();
+    public ItemHarvestEvent(@NotNull Craft craft, List<ItemStack> harvested) {
```

Comment:
Perhaps we should also add the type of block harvested and the location (or just a block object)?